packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/new.module.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
+import 'ovh-api-services';
+import 'ovh-ui-angular';
+
+import component from './new.component';
+import offers from '../components/offers';
+import offerDetails from '../components/offer-details';
+import routing from './new.routing';
+import termsOfUse from '../components/termsOfUse';
+
+const moduleName = 'ovhManagerDedicatedCloudBackupNewModule';
+
+angular
+  .module(moduleName, [
+    offers,
```

Comment:
No translations, no ui-router, no ui-kit ?

Reply:
Added the required dependencies